### PR TITLE
Recomended labels

### LINKED ISSUE #458: Use recommended kubernetes and openshift labels.

### PR DESCRIPTION
**Pull request by iocanel**
Fixes #458 

The pull request also addresses ordering issues of decorators.

### CODE REVIEW

File: core/src/main/java/io/dekorate/kubernetes/decorator/RemoveLabelDecorator.java [hunk 1]
```diff
@@ -0,0 +1,77 @@
...
+ * limitations under the License.
+ */
+package io.dekorate.kubernetes.decorator;
+
+import io.dekorate.deps.kubernetes.api.model.ObjectMeta;
+import io.dekorate.deps.kubernetes.api.model.ObjectMetaBuilder;
+import io.dekorate.doc.Description;
+
+/**
+ * A decorator that adds a label to resources.
```
Comment (geoand) [inline core/src/main/java/io/dekorate/kubernetes/decorator/RemoveLabelDecorator.java]: I assume you mean removes here?

Comment (iocanel) [reply core/src/main/java/io/dekorate/kubernetes/decorator/RemoveLabelDecorator.java]: right

**Comment by iocanel** [comment]
@geoand: A lot has changed since you reviewed. So you may wanna have a seconf look.

File: assets/config.md [hunk 2]
```diff
@@ -274,7 +274,7 @@ The section below describes all the available subtypes.
 
 | Property                                         | Type                          | Description | Default Value |
 |--------------------------------------------------|-------------------------------|-------------|---------------|
-| dekorate.knative.group                           | String                        |             |               |
+| dekorate.knative.part-o                          | String                        |             |               |
```
Comment (geoand) [inline assets/config.md]: `part-of` I believe

**Pull request merged**